### DISCUSSION TITLE
[CONTINUED] Users edit profile details on site - not in CP

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#4674] [Q&A]
This is a CONTINUATION from the thread in the old forum (I thought it would be helpful to add a link, and tell it in the new subject):

[https://statamic.com/forum/5511-users-edit-profile-details-on-site-not-in-cp](url)

Are there any plans to add a `{{ user:profile_form }}` tag to V3?

Or is there some addon author activate on such a tag?

Or has someone some cool ideas to do that?

We need this kind of solution and I wonder why it isn't there. If we have users, they should have a chance to edit their profile (if useful inside an app, of course) ...

Best, Jens Olaf

**Comment by edalzell** [comment]
It looks like this addon might do it @jensolafkoch: https://github.com/jacksleight/statamic-members, why don't you give it a try?

**Reply by jensolafkoch** [reply]
Yes, I had seen this one briefly before (and it looks like it "has it all") but the problem was that we already had most of the other member functionality built in on our own and just wanted to add profile editing. For the short term I did so with a new Laravel route and controller (as in the link in my original post) ...

But I guess Jack's addon would be the easier to go forward and maybe we switch to it.
